### ISSUE TITLE
Standard bidCpmAdjustment not working

### ISSUE THREAD

**Issue by dugwood** [prebid/Prebid.js#1628]
## Type of issue
`bidCpmAdjustment` set on `standard` doesn't work.

## Description
In the documentation: http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.bidderSettings
=> bidCpmAdjustment is either for `standard` or `adapter-specific`

But:
https://github.com/prebid/Prebid.js/blob/master/src/bidmanager.js#L434
=> here we only look for the `bidderCode` (adapter-specific) function

And:
https://github.com/prebid/Prebid.js/blob/master/src/bidmanager.js#L453
=> here we never set the function if not defined

## Steps to reproduce
Set:
```
pbjs.bidderSettings = {
    standard: {
        bidCpmAdjustment: function (bidCpm, bid) { console.log(bidCpm); return bidCpm; }
    }
};
```

### Expected results
The console should log the bidCpm value

### Actual results
No log (so no adjustment)

## Platform details
Prebid: 0.29.0

**Comment by mkendall07** [comment]
@dugwood thanks for the report. This has always been the case, the documentation is wrong. What's the use case to adjust all the bids?

**Comment by dugwood** [comment]
Thanks @mkendall07. I should fix the documentation :-)

So far I was using it with a common function, that logs (in a debug mode) and adjust CPMs according to another external setting. For example:
```
grossToNet: {
		appnexusAst: 0.9, // value isn't the real one, just to illustrate
		rubicon: 0.95,
		indexExchange: 0.85
}
```

Now that all plaforms are sending the net revenue, it's indeed useless. So far I was looping on all bidders to set the `bidCpmAdjustment` (for every call, that was not that great), and just found out about the `standard` property. So I just tried to use it :-)

Although I could use the event, I still need to set the bidCpm to zero if it's under a predefined CPM, which is set on the platform (APN, ...) but also in our code because some platform don't have minimum CPM settings.

So a `standard` setting would be great. But I can do as before, setting it to the same function for every bidder (once and for all, not for each auction).

**Comment by mkendall07** [comment]
I can see how this is more maintainable than setting individual bidder settings. We accept PRs :)

**Comment by dugwood** [comment]
@mkendall07 it's done :-) Had a difficult time running tests (sooooo long!), but in the end I think it's better that the function is implemented than the documentation changed.

**Comment by jaiminpanchal27** [comment]
Fixed in #1645.
